hazelcast/src/test/java/com/hazelcast/test/starter/test/HazelcastVersionLocatorTest.java

```diff
@@ -19,64 +19,54 @@
 import static com.hazelcast.test.starter.HazelcastVersionLocator.Artifact.EE_JAR;
 import static com.hazelcast.test.starter.HazelcastVersionLocator.Artifact.OS_JAR;
 import static com.hazelcast.test.starter.HazelcastVersionLocator.Artifact.OS_TEST_JAR;
-import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
-import org.junit.BeforeClass;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized.Parameter;
-import org.junit.runners.Parameterized.Parameters;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
 import org.junit.runners.Parameterized.UseParametersRunnerFactory;
```

Comment:
I think there is one JUnit4 and JUnit5 mix here

Reply:
23474563f241fc4939b6278d771574dcaa96019b